create_student.rb

```diff
@@ -0,0 +1,26 @@
+require './student'
+require './person'
+require './nameable'
+
+def create_a_student
+  print 'Age: '
+  age = gets.chomp
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Has parent permission? [Y/N]: '
+  permission = gets.chomp
+
+  case permission
+  when 'y', 'Y'
+    @persons.push(Student.new(true, age, name))
+    puts 'Student created successfully'
+  when 'n', 'N'
+    @persons.push(Student.new(false, age, name))
+    puts 'Student created successfully'
+  else
+    puts 'Invalid permission (Enter Y/N)'
+    library_menu
+  end
+end
```

Comment:
- For the student, I don't seem to be able to create a student also. Please, fix the argument error it throws from the terminal below:


![Screenshot from 2023-01-19 08-56-17](https://user-images.githubusercontent.com/59397159/213386174-a680a838-a1d5-4960-9e64-fefc56bb80e4.png)